skflow/__init__.py

```diff
@@ -28,7 +28,7 @@
 from skflow.trainer import TensorFlowTrainer
 from skflow import models, data_feeder
 from skflow import preprocessing
-
+from skflow.io.pandas_io import *
```

Comment:
what about just `from skflow.io import *` ? also do we want to just drop everything in skflow namespace? maybe instead `from skflow import io`?